packages/tooltip/src/Tooltip.tsx

```diff
@@ -11,26 +11,57 @@ import { useMergeRefs } from '@lidofinance/hooks'
 import { isElement } from 'react-is'
 import { TooltipProps } from './types'
 
+const KEEP_TIMEOUT_DEFAULT = 150
```

Comment:
It is possible to consider a variant without complex logic of timeouts. For example, hide the popup always after a certain time. And check what the mouse pointer is hovering over globally. And if it's a popup, don't hide it.
It seems to me that the current logic is not very convenient to use.

Reply:
I do not see reason to use global mouse pointer events if we make it work without it.